extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -231,6 +234,10 @@ public static function onWikiaMobileAssetsPackages( array &$jsStaticPackages, ar
 			array_splice( $jsStaticPackages, $coreGroupIndex, 0, self::ASSET_GROUP_ADENGINE_TABOOLA );
 		}
 
+		if ($wgEnableKruxOnMobile === true) {
```

Comment:
because it's a boolean, and it's filtered to be boolean here https://github.com/Wikia/app/blob/aden-1560-enable-krux-on-mobile/extensions/wikia/AdEngine/AdEngine2Hooks.class.php#L46

Also previous two comparisons are also strict in this function